crates/nu-cmd-lang/src/core_commands/do_.rs

```diff
@@ -37,11 +36,6 @@ impl Command for Do {
                 "ignore external program errors as the closure runs",
                 Some('p'),
             )
-            .switch(
```

Comment:
In case we do decide to keep the exit codes as `ShellError`, I think you need to change this to a deprecated flag rather than removing it. I have `do -c` in my config file and it fails to parse